### PR TITLE
Fixes #1028:Infobox now changes color according to Theme

### LINKED ISSUE #1028: Infobox do not changes color according to theme.
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

Infobox do not changes color according to theme.
**Expected behavior:**

It should change color according to different themes
**Screenshots of the issue:**

image

### PR DESCRIPTION
**Pull request by praveenojha33**
Fixes #1028 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:
Infobox now changes color according to Theme
Surge Link:https://pr-1029-fossasia-susper.surge.sh
Screen Shot:
image

### CODE REVIEW

File: src/app/infobox/infobox.component.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import {Component, OnInit, ChangeDetectorRef} from '@angular/core';
```
Comment (simsausaurabh) [inline src/app/infobox/infobox.component.ts]: ```ChangeDetectorRef``` is also not required if you are removing it in constructor.

File: src/app/infobox/infobox.component.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-import {Router, ActivatedRoute} from '@angular/router';
 import {Store} from '@ngrx/store';
```
Comment (simsausaurabh) [inline src/app/infobox/infobox.component.ts]: ```Store``` is also not required if you are removing it in constructor.

**Comment by simsausaurabh** [comment]
The second one is not prior (could be discussed later) but the first one to fix footer color alignment could be fixed in this same patch since the changes are minimal and the patch references the theme fix. I hope it works for you.

**Comment by praveenojha33** [comment]
@simsausaurabh All changes have been made please review now.

**Pull request merged**